spec/forms/openid_connect_token_form_spec.rb

```diff
@@ -403,7 +403,7 @@
         expect(submission.to_h).to include(
           success: false,
           errors: form.errors.messages,
-          error_details: hash_including(*form.errors.keys),
+          error_details: hash_including(*form.errors.attribute_names),
```

Comment:
> This change is for ...? The deprecation warning? I hadn't noticed that previously. Should we handle that separately from these changes? I could make a pass at it, since I was the one who'd introduced them in #5048.

On closer look, this should be the only place we need to update, because in other cases, the `errors.keys` we're referencing in `hash_including` refers to a hash, where `Hash#keys` is fine.

Fine to keep imo 👍

Reply:
yep, that was the deprecation 🙂